src/inference/include/openvino/runtime/properties.hpp

```diff
@@ -602,6 +602,59 @@ static constexpr Property<uint64_t, PropertyMutability::RW> dynamic_quantization
  */
 static constexpr Property<element::Type, PropertyMutability::RW> kv_cache_precision{"KV_CACHE_PRECISION"};
 
+/**
+ * @brief Enum to define possible cache quant schema hints.
+ */
+enum class CacheQuantMode { AUTO = 0, BY_CHANNEL = 1, BY_TOKEN = 2 };
+
+/** @cond INTERNAL */
+inline std::ostream& operator<<(std::ostream& os, const CacheQuantMode& mode) {
+    switch (mode) {
+    case CacheQuantMode::AUTO:
+        return os << "AUTO";
+    case CacheQuantMode::BY_CHANNEL:
+        return os << "BY_CHANNEL";
+    case CacheQuantMode::BY_TOKEN:
+        return os << "BY_TOKEN";
```

Comment:
@alexsu52  Hi Alexander, I was told that NNCF has also some naming convention regarding kv cache compression.  Do you have any opinion for categorizing & naming these? @sshlyapn

Reply:
Hi @yeonbok. We are using the naming which was introduced in this PR.